packages/jest-config/src/readConfigFileAndSetRootDir.ts

```diff
@@ -26,7 +27,7 @@ import {
 export default async function readConfigFileAndSetRootDir(
   configPath: string,
 ): Promise<Config.InitialOptions> {
-  const isTS = configPath.endsWith(JEST_CONFIG_EXT_TS);
+  const isTS = configPath.endsWith(JEST_CONFIG_EXT_TS || JEST_CONFIG_EXT_CTS);
```

Comment:
```suggestion
  const isTS =
    configPath.endsWith(JEST_CONFIG_EXT_TS) ||
    configPath.endsWith(JEST_CONFIG_EXT_CTS);
```

@DerTimonius I think this is your issue - `cts` was never considered

Reply:
oh man. you're right 🙈 
implemented this change with the last commit and fixed the tests (all running locally! they did not before, so this is a huge improvement)